packages/lib-utils/src/components/details-page-header/utils/Breadcrumbs.test.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { render, screen, fireEvent } from '@testing-library/react';
+import React from 'react';
+import { MemoryRouter, Routes, Route } from 'react-router-dom';
+import '@testing-library/jest-dom';
+import type { BreadcrumbsProps } from './Breadcrumbs';
+import { Breadcrumbs } from './Breadcrumbs';
+
+const mockProps: BreadcrumbsProps = {
+  breadcrumbs: [
+    { name: 'Workspaces', path: '/workspaces' },
+    { name: 'Workspace details', path: '/workspaces/demo-workspace' },
+  ],
+};
+
+const breadcrumbsJSX = (args: BreadcrumbsProps) => (
+  <MemoryRouter initialEntries={['/workspaces/demo-workspace']}>
+    <Routes>
+      <Route element={<Breadcrumbs {...args} />} path="/workspaces/demo-workspace" />
+      <Route element={<div>Workspaces List Page</div>} path="/workspaces" />
+    </Routes>
+  </MemoryRouter>
+);
+
+describe('Breadcrumbs', () => {
+  test('Breadcrumbs are rendered', () => {
+    render(breadcrumbsJSX(mockProps));
+
+    expect(screen.getByText('Workspaces')).toBeVisible();
```

Comment:
👍 for tests! One thing to note - this test will eventually break when we add i18n. I usually favor searching for DOM classes/ID's before text for this reason (unless the test is specifically testing the text itself).